### PR TITLE
Add account to permanent_account_key

### PR DESCRIPTION
**Pull request by Raphexion**

### CODE REVIEW

File: lib/wise_homex/models/account.ex [hunk 1]
```diff
@@ -7,6 +7,7 @@ defmodule WiseHomex.Account do
     belongs_to :owner, __MODULE__
     has_many :admin_integrations, WiseHomex.AdminIntegration, foreign_key: :admin_id
     has_many :account_users, WiseHomex.AccountUser
+    has_many :permanent_account_keys, WiseHomex.PermanentAccountKeys
```
Comment (lasseebert) [inline lib/wise_homex/models/account.ex:10]: This relation is not exposed by the API, so it should probably be deleted here.

**Review by lasseebert — CHANGES_REQUESTED** [review]
Good :+1: See my one comment.

File: lib/wise_homex/models/account.ex (hunk 1 above)
Comment (Raphexion) [reply lib/wise_homex/models/account.ex]: Removed :+1: (I just added so that the association was complete) :sunny:

**Comment by lasseebert** [comment]
@Raphexion Heads up: CI fails.

**Review by lasseebert — CHANGES_REQUESTED** [review]
The CI fails, probably because of the upgrade to Elixir 1.13.0. Perhaps make the elixir and erlang updates in a separate PR, so that it is easy to see what changed, if it needs code changes.

**Comment by Raphexion** [comment]
@lasseebert I actually added the elixir/erlang changes to make the PR pass CI. Because it broke with the original commit.

**Comment by Raphexion** [comment]
Screenshot from 2021-12-15 08-33-56

**Comment by Raphexion** [comment]
@lasseebert Now the same commit passes CI (it failed yesterday even when I tried to re-run it multiple times).

**Comment by lasseebert** [comment]
@Raphexion My guess is that it has something to do with the caching of `deps` and `_build`

**Comment by Raphexion** [comment]
I think you are correct. It was so strange because most of the "matrix" passed. But the error mentioned something about ssl.

Screenshot from 2021-12-15 08-38-05

**Review by lasseebert — APPROVED** [review]
:+1:

I'm not sure what happened with the CI errors, but probably related to caching.

**Pull request merged**